agent/lib/qdr.js

```diff
@@ -42,6 +44,7 @@ var Router = function (connection, router, agent) {
         this.tracking = {
             sent: 0,
             recv: 0,
+            outstanding: 0,
```

Comment:
Is it worth adding outstanding to the info reported periodically by Router.prototype.log_info?